scripts/build.js

```diff
@@ -29,8 +29,8 @@ const dist = "dist";
     }
 
     for (const f of await fs.readdir(src)) {
-      if (/\.(?:js|css|html)$/.test(f)) {
-        await fs.copyFile(path.join(src, f), path.join(dist, f));
+      if (/\.(?:js|css|html)$/.test(f) || f === "assets") {
+        await fs.cp(path.join(src, f), path.join(dist, f), {recursive: true});
```

Comment:
I got a warning on this saying, "UnhandledPromiseRejectionWarning: TypeError: fs.cp is not a function"

Reply:
I mentioned this in discord, but I was hoping you had Node >= 16.7.0 per [this SO answer](https://stackoverflow.com/a/69807672/6243352). Apparently copying a directory of files recursively is a bit awkward in prior versions, but it can be done, so if you don't want to upgrade that's OK, I'll downgrade the code.